src/navigations/HomeBottomTabs.js

```diff
@@ -29,7 +33,7 @@ function HomeBottomTabs({navigation}) {
   const initialStartup = useRecoilValue(InitialStartupAtom);
   const otherProfileData = useRecoilValue(otherProfileAtom);
   const [, setProfileAtom] = useRecoilState(profileAtom);
-  const {totalUnreadCount} = useRootChannelListHook();
+  const {signedChannelUnreadCount, anonymousChannelUnreadCount} = useRootChannelListHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It seems like you've replaced `totalUnreadCount` with `signedChannelUnreadCount` and `anonymousChannelUnreadCount`. Make sure that this change doesn't affect other parts of your application where `totalUnreadCount` might be used.



<!-- This is an auto-generated comment by OSS CodeRabbit -->